.github/workflows/build_and_push_image.yml

```diff
@@ -0,0 +1,83 @@
+name: Build and push narrative images
+
+on:
+  workflow_call:
```

Comment:
This workflow builds the narrative and narrative_version images; it is called by a parent workflow.